api/app/Http/Controllers/EntityController.php

```diff
@@ -91,15 +106,15 @@ public function postOne(Request $request)
      */
     public function putOne($id, Request $request)
     {
-        $this->validateId($id, $this->getKeyName(), $this->validateRequestRule);
+        $this->validateId($id, $this->getModel()->getKeyName(), $this->validateRequestRule);
 
         try {
-            $model = $this->getRepository()->find($id);
+            $model = $this->getEntityById($id);
```

Comment:
From memory, it didn't really make sense in the context of a repository. It makes sense for a controller though.